sites/partners/src/applications/PaperApplicationForm/FormTypes.ts

```diff
@@ -57,8 +56,8 @@ export type ApplicationTypes = {
     phoneNumberType?: string
   }
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  preferences: Record<string, any>
-  programs: Record<string, string>
+  preferences?: Record<string, any>
+  programs?: Record<string, string>
```

Comment:
should'n't these both be the same type now and can we explicitly type it?

Reply:
They should definitely be the same! This is representing the react-hook-form structure, which is super wonky and difference between checkboxes and radios, so I'll update the type but it will still be pretty open.